src/components/dashboard/DashboardPage.jsx

```diff
@@ -11,39 +11,23 @@ import { LoadingSpinner } from '../loading-spinner';
 import { DashboardMainContent } from './main-content';
 import { DashboardSidebar } from './sidebar';
 import Hero from './Hero';
+import NotFoundPage from '../NotFoundPage';
```

Comment:
nit: for import consistency, I've been trying to keep imports that go up a directory prior to imports from the same directory, i.e. `NotFoundPage` import after `LoadingSpinner`.